rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -60,6 +60,21 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalResize(RocalContext context, RocalIm
                                                    unsigned resize_longer = 0,
                                                    RocalResizeInterpolationType interpolation_type = ROCAL_LINEAR_INTERPOLATION);
 
+/// Accepts U8 and RGB24 input.
+/// \param context
+/// \param input
+/// \param dest_width
+/// \param dest_height
+/// \param mean
+/// \param std_dev
```

Comment:
@SundarRajan28 Need a sentence explaining these. Refer to other functions

Reply:
Added definitions for each parameter passed to the RMN augmentation function